cypress/integration/firstTests.spec.js

```diff
@@ -1,7 +1,42 @@
 /// <reference types="cypress" />
 
-describe('', () => {
-  it('', () => {
-    
+beforeEach(() => {
+  cy.visit('https://react-redux.realworld.io')
+})
+
+describe('First Test', () => {
+  it('Sign In with valid data', () => {
+  cy.get (':nth-child(2) > .nav-link')
+    .click()
+    .get (':nth-child(1) > .form-control')
+    .type ('newuser2022@gmail.com')
```

Comment:
```suggestion
    cy.get (':nth-child(1) > .form-control')
      .type ('newuser2022@gmail.com');
```
якщо писати одним ланцюгом - працювати буде, але зазвичай так не пишуть.
після тайп та клік крапка з комою і далі нові команди